src/utils/format-numbers.js

```diff
@@ -27,19 +27,10 @@ export function formatNumber(
   significantDigits: number = 2,
   maxFractionalDigits: number = 3
 ): string {
-  /*
-   * Note that numDigitsOnLeft can be negative when the first non-zero digit
-   * is on the right of the decimal point.  0.01 = -1
-   */
-  const numDigitsOnLeft = Math.floor(Math.log10(Math.abs(value))) + 1;
-  let places = significantDigits - numDigitsOnLeft;
-  if (places < 0) {
-    places = 0;
-  } else if (places > maxFractionalDigits) {
-    places = maxFractionalDigits;
-  }
-
-  return value.toFixed(places);
+  return value.toLocaleString(undefined, {
+    maximumSignificantDigits: significantDigits,
+    maximumFractionDigits: maxFractionalDigits,
+  });
 }
 
 export function formatPercent(value: number): string {
```

Comment:
you can also change this function, and use `{ style: 'percent' }` in the second argument of `toLocaleString`.

Then you can use this in `src/profile-logic/call-tree.js` to format the %age value as well.

Reply:
Adding only `{ style:'percent' }` is not giving correct results.

Reply:
can you tell m what you tried and what this yields? Thanks!

Reply:
```export function formatPercent(value: number): string {
  return value.toLocaleString(undefined, {
    style:'percent'
  });
}```  This is not giving percentage in one point decimal

Reply:
Of course not, because you need to give the right arguments to `toLocaleString`...

Reply:
```return value.toLocaleString(undefined, {
    maximumFractionDigits: 1,
    style:'percent'
  });``` is giving one point decimal but when we divide it should not give percentage in decimal

Reply:
can you please give me an example so that I can follow you better?

Reply:
```<div className="tooltipLabel">
              Cells evicted:
            </div>
    -       15.9K / 26.6K (60%)
    +       15.9K / 26.6K (59.6%)```

Reply:
`minimumFractionalDigits` seems to work